administrator/components/com_j2store/layouts/list/default_filters.php

```diff
@@ -1,54 +1,113 @@
 <?php
-defined('_JEXEC') or die;
+/**
+ * @package J2Store
+ * @copyright Copyright (c)2014-17 Ramesh Elamathi / J2Store.org
+ * @copyright Copyright (c) 2024 J2Commerce . All rights reserved.
+ * @license GNU GPL v3 or later
+ */
+
+
+// No direct access to this file
+defined('_JEXEC') or die('Restricted access');
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+
+HTMLHelper::_('bootstrap.collapse', '[data-bs-toggle="collapse"]');
 $row_class = 'row';
 $col_class = 'col-md-';
 if (version_compare(JVERSION, '3.99.99', 'lt')) {
     $row_class = 'row-fluid';
     $col_class = 'span';
 }
+
+$count = 0;
+$count2 = 0;
+$fieldsearchableCount = 0;
+
 ?>
 <?php if(isset($vars->header) && !empty($vars->header)):?>
-    <div class="<?php echo $row_class?>">
-        <div class="<?php echo $col_class;?>12">
-            <?php
-            $sortable_field = array();
-            //Search field
-            foreach ($vars->header as $name => $field): ?>
-                <?php if(isset($field['sortable']) && $field['sortable'] === 'true'){
-                    $sortable_field[$name] = JText::_($field['label']);
-                }; ?>
-                <?php if(isset($field['type']) && $field['type'] === 'fieldsearchable'):?>
-                <div class="pull-left searchable_field">
-                    <input id="search_<?php echo $name;?>" type="text" name="<?php echo $name;?>" value="<?php echo $vars->state->get($name,'');?>" placeholder="<?php echo JText::_($field['label'])?>">
-                    <a class="btn" onclick="document.adminForm.submit()"><i class="icon-search"></i></a>
-                    <a class="btn" onclick="document.adminForm.<?php echo $name;?>.value='';document.adminForm.submit()"><i class="icon-remove"></i></a>
-                </div>
+	<?php $sortable_field = array();?>
+    <div class="btn-toolbar w-100 justify-content-end mb-3">
+        <?php foreach ($vars->header as $name => $field):
+
+            if(isset($field['sortable']) && $field['sortable'] === 'true'){
+                $sortable_field[$name] = Text::_($field['label']);
+            } ?>
+
+            <?php if(isset($field['type']) && $field['type'] === 'fieldsearchable'):?>
+                <?php $count++;?>
+                <?php $fieldsearchableCount++;?>
+                <?php if($count == 1) : ?>
+                    <div class="filter-search-bar btn-group flex-grow-1 flex-lg-grow-0 mb-2 mb-lg-0">
+                        <div class="input-group w-100 searchable_field">
+                            <input id="search_<?php echo $name;?>" type="text" name="<?php echo $name;?>" value="<?php echo $vars->state->get($name,'');?>" placeholder="<?php echo Text::_($field['label'])?>" class="form-control j2store-product-filters">
+                            <span class="filter-search-bar__label visually-hidden">
+                                <label id="search-lbl" for="search"><?php echo Text::_($field['label'])?></label>
```

Comment:
the id of the input is search_$name on line 43
```suggestion
                                <label id="search-lbl" for="search_<?php echo $name;?>"><?php echo Text::_($field['label'])?></label>
```